pipeline.go

```diff
@@ -176,14 +166,43 @@ func (p *Pipeline) Run() (PipelineResult, error) {
 	return result, nil
 }
 
-func storeError(msg string, err error) error {
-	return fmt.Errorf("%s: %s", msg, err)
-	// TODO: Store error
-	//er.Cr.AgencyID = filepath.Base(job)
-	//Store Error
-	//Build(storeErrDir, commit, conf)
-	//execStoreErr(er, conf)
-	//continue
+func errorHandler(result *PipelineResult, ser StageExecutionResult, previousStatus status.Code, msg string, handler Stage) (PipelineResult, error) {
+	ser.FinalTime = time.Now()
+	result.StagesResults = append(result.StagesResults, ser)
+
+	if handler.Dir != "" {
+		id := fmt.Sprintf("%s/%s - Error Handler", result.Name, ser.Stage)
+		buildResult, err := buildImage(id, handler.Dir, handler.BuildEnv)
+		if err != nil {
+			result.Status = status.ErrorHandlerError
+			result.FinalTime = time.Now()
+			return *result, fmt.Errorf("error when building image: %s", err)
```

Comment:
Ser mais específica na mensagem de erro, dizendo especificando que é o error handler?

Reply:
Ajuste realizado.
`("error when building image for error handler: %s", err)`...